cmd/clair-load-test/reports.go

```diff
@@ -132,20 +133,24 @@ func NewConfig(c *cli.Context) *TestConfig {
 	}
 }
 
+// fetchLayers calculates the layers number used to fetch images.
+// It returns an integer indicating amount of layers.
+func fetchLayers(ctx context.Context, layers int, validLayers []int) int {
```

Comment:
I think this function is poorly named, it's just returning the number of layer per image to pull, not fetching anything.

Reply:
renamed it as `calculateLayers`.